ncm-network/src/main/pan/components/network/config-nmstate.pan

```diff
@@ -0,0 +1,14 @@
+# ${license-info}
+# ${developer-info}
+# ${author-info}
+
+unique template components/network/config-nmstate;
+
+include 'components/network/config';
+
+prefix "/software/components/network";
+"ncm-module" = "nmstate";
+
+# Add dependency that can't be added to rpm directly
+prefix '/software/packages';
+'nmstate' = dict();
```

Comment:
you want to add `NetworkManager-config-server` here, and possibly in the early list of kickstart rpms, like
```
prefix "/system/aii/osinstall/ks";
"NetworkManager-config-server" = dict();
```

Reply:
mask the nmstate service
```
"/software/components/systemd/unit/nmstate/state" = "masked";
```
(or something like that).

Reply:
@stdweird configuring another component in the context of a component configuration is quite unusual... At the same time I understand that it is a requirement and is probably better done here that somewhere in the template library. A site may use this component without using the template library or using an outdated version of it... May be worth a comment at least...

Reply:
@jouvin i don't follow. the relevant comment is there: this is for rpm dependencies that we can set in the component